packages/node_modules/@webex/plugin-meetings/src/media/properties.js

```diff
@@ -61,6 +61,14 @@ export default class MediaProperties {
   }
 
   setRemoteStream(remoteStream) {
+    const stream = this.remoteStream;
+    const isCompatible = stream && stream.getTracks && stream.addTrack;
+
+    if (isCompatible) {
+      // Combine audio and video tracks into one stream
+      stream.getTracks().forEach((track) => remoteStream.addTrack(track));
```

Comment:
@anthonywebb why do we need to add tracks here !? you can use the media:ready event to save the stream,